gamestonk_terminal/reports/stored/20211102_161336_econ_data.md

```diff
@@ -0,0 +1,90 @@
+# Economy Data - 2021/11/02 16:16:06
```

Comment:
You shouldn't be committing these files. That's why test is failing.

Since you are at it, can you add this folder to gitignore?